src/lib/viewers/controls/media/MediaSettings.tsx

```diff
@@ -1,29 +1,68 @@
 import React from 'react';
+import noop from 'lodash/noop';
+import getLanguageName from '../../../lang';
+import MediaSettingsMenuAudioTracks, { AudioTrack, Props as AudioTracksProps } from './MediaSettingsAudioTracks';
 import MediaSettingsMenuAutoplay, { Props as AutoplayProps } from './MediaSettingsMenuAutoplay';
 import MediaSettingsMenuRate, { Props as RateProps } from './MediaSettingsMenuRate';
 import Settings, { Menu } from '../settings';
 
-export type Props = AutoplayProps & RateProps & { className?: string };
+export type Props = Partial<AudioTracksProps> & AutoplayProps & RateProps & { className?: string };
+
+const generateAudioTrackLabel = (language: string, index: number): string => {
+    let label = `${__('track')} ${index + 1}`;
+    if (language !== 'und') {
+        label = `${label} (${getLanguageName(language) || language})`;
+    }
+
+    return label;
+};
+
+const addLabels = (audioTracks: Array<AudioTrack>): Array<AudioTrack> =>
+    audioTracks.map((track, index) => {
+        const { language } = track;
+        const label = generateAudioTrackLabel(language, index);
+        return {
+            ...track,
+            label,
+        };
+    });
 
 export default function MediaSettings({
+    audioTrack,
+    audioTracks = [],
     autoplay,
     className,
+    onAudioTrackChange = noop,
     onAutoplayChange,
     onRateChange,
     rate,
 }: Props): JSX.Element {
     const autoValue = autoplay ? __('media_autoplay_enabled') : __('media_autoplay_disabled');
     const rateValue = rate === '1.0' || !rate ? __('media_speed_normal') : rate;
+    const labelledAudioTracks = React.useMemo(() => addLabels(audioTracks), [audioTracks]);
```

Comment:
I feel like this could probably be simplified, but we can go with it for now. We may need to revisit as we add more menus for Dash, since this component will probably get large quickly.